rbtcwrapperproxy/contracts/RBTCWrapperProxy.sol

```diff
@@ -124,13 +153,15 @@ contract RBTCWrapperProxy is ContractRegistryClient {
      * 4.Transfers pool tokens to user
```

Comment:
the function sends pool tokens to `liquidityMiningContract` instead

same for `addLiquidityToV1`